target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -253,6 +253,14 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth1"
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth0"
```

Comment:
You still have eth1 and eth0 exchanged here compared to your network setup.

Reply:
Not sure I understand. eth0/1 has been reversed recently (for very obscur reason), so why suddently this is now against reversed back ?

Reply:
Currently, you are not setting anything in 02_network, so will use the default case there:
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/base-files/etc/board.d/02_network#L294-L295
`ucidef_set_interfaces_lan_wan "eth0" "eth1"`

This will mean that on your _running_ device, eth0 (as in ip l/ifconfig) will be lan and eth1 will be wan. In 01_leds, eth1 and eth0 will be the same as in 02_network, thus "eth1" should be wan and "eth0" should be lan.

So, what you do in 01_leds should always match what you do in 02_network.

What you are referring about with "eth0/1 has been reversed recently" is a different topic. Here, the link between &eth0 and &eth1 in DTS and the eth0 and eth1 on running device is concerned. If you want to introduce changes due to that, you could use another setup in 02_network. But then 01_leds still would have to be consistent with 02_network.

Reply:
If you switch eth0/1 by default, why erasing this chnage in 02_network ?

Anyway this nonsense does not matter to me. Can you just state how it shall be in 01_lets and 02_network ?

Reply:
Well, that depends on what you have with the stock firmware. Are the ports (!) on the running device currently assigned the same way (lan=lan, wan=wan) as on the device with stock firmware?
Depending on your "yes" or "no" here I can reply with further hints.

Reply:
So, you have changed it, but what about "yes" or "no"?